web/src/store/index.ts

```diff
@@ -99,30 +108,30 @@ const appstate = {
 };
 
 const getters = {
-  dataset: state => id => state.datasets[id],
-  ready: state => id => state.datasets[id] && state.datasets[id].ready,
+  dataset: (state: any) => (id: string) => state.datasets[id],
```

Comment:
I've never used typescript in vuex, but I've always though it looked perilous.  Are you following any particular guide?  Could I read it?

Reply:
Do you mind clarifying why using typescript with vuex would be dangerous?

I'm only using a styleguide implicitly, via [TSLint](https://github.com/palantir/tslint), which was set up by Vue CLI's config for typescript.

(In this particular case, the forced parenthesization of a single argument in an arrow function is actually justified: typescript accepts `x => ...`, `(x) => ...`, and `(x: T) => ...`, but not `x: T => ...`.)